web/ui_utils.js

```diff
@@ -187,8 +187,8 @@ function watchScroll(viewAreaElement, callback) {
 function parseQueryString(query) {
   const parts = query.split("&");
   const params = new Map();
-  for (let i = 0, ii = parts.length; i < ii; ++i) {
-    const param = parts[i].split("="),
+  for (const part of parts) {
```

Comment:
Nit: You could probably just do `for (const part of query.split("&")) {` now, if you wanted to, and remove the separate variable.